src/EditorFeatures/TestUtilities/AbstractCommandHandlerTestState.cs

```diff
@@ -265,8 +265,8 @@ public ITextSnapshotLine GetLineFromCurrentCaretPosition()
 
         public string GetLineTextFromCaretPosition()
         {
-            var caretPosition = Workspace.Documents.Single(d => d.CursorPosition.HasValue).CursorPosition.Value;
-            return SubjectBuffer.CurrentSnapshot.GetLineFromPosition(caretPosition).GetText();
+            var caretPosition = GetCaretPoint();
```

Comment:
so... now we are getting caret position from the text view? any reason we changed to this way?